web/password_prompt.js

```diff
@@ -33,12 +33,12 @@ import { PasswordResponses } from './pdfjs';
 /**
  * @class
  */
-var PasswordPrompt = (function PasswordPromptClosure() {
+class PasswordPrompt {
   /**
    * @constructs PasswordPrompt
```

Comment:
Similarly, since the thing below is now a `constructor`, I think that this line is now superfluous.